api/src/main/java/org/pmiops/workbench/calhoun/CalhounRetryHandler.java

```diff
@@ -51,14 +50,16 @@ protected void logNoRetry(Throwable t, int responseCode) {
   }
 
   @Autowired
-  public CalhounRetryHandler(
-      BackOffPolicy backoffPolicy, Provider<TermsOfServiceApi> termsOfServiceApiProvider) {
-    super(backoffPolicy, new CalhounRetryPolicy(), termsOfServiceApiProvider);
+  public CalhounRetryHandler(BackOffPolicy backoffPolicy, FireCloudService fireCloudService) {
+    super(
+        backoffPolicy,
+        new CalhounRetryPolicy(),
+        fireCloudService,
+        ExceptionUtils::convertCalhounException);
```

Comment:
opportunistic change while I was changing the signature anyway, which allowed making the code more consistent